integration_tests/gateway/package.json

```diff
@@ -9,9 +9,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@apollo/gateway": "^0.8.1",
-    "apollo-server": "^2.8.1",
-    "graphql": "^14.4.2"
+    "@apollo/gateway": "^2.0.5",
```

Comment:
Nice that you made it work with latest, I went with the same for Cutest, almost working there too (it's a bit more complex because we don't use apollo server but express instead)